mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -3,6 +3,8 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	FIELD field_16762 game Lnet/minecraft/class_3799;
 	FIELD field_1686 windowProvider Lnet/minecraft/class_3682;
 	FIELD field_1687 world Lnet/minecraft/class_638;
+		COMMENT Represents the world the client is currently viewing.
+		COMMENT This field is not null when in game.
```

Comment:
Should say "connected to a logical server" than "in game"

Reply:
I think this is fine, this field in particular only says "currently, we're viewing _THIS_ world". It doesn't really say anything about a server connection in my opinion.